commands/Tools/dbl.js

```diff
@@ -0,0 +1,43 @@
+const { Command } = require('klasa');
+const { MessageAttachment } = require('discord.js');
+const fetch = require('node-fetch');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			cooldown: 15,
+			usage: '<target:user>',
+			usageDelim: ''
+		});
+	}
+
+	async run(msg, [target]) {
+		let url = '';
+		if (!target.bot) throw 'You need to provide a bot id and not a user one silly';
+		const type = msg.flags.type ? String(msg.flags.type) : 'large';
+		const icon = msg.flags.icon ? '?noavatar=true' : '';
+		switch (type) {
```

Comment:
Can refactored to ``https://discordbots.org/api/widget${type ? `/${type}` : ''}/${target.id}.png${icon}``, and make type default to null/empty string